ports/catwarrior/Cat Warrior.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+# Pm
+source $controlfolder/control.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+source $controlfolder/tasksetter
+
+# Variables
+GAMEDIR="/$directory/ports/catwarrior"
+cd "$GAMEDIR/gamedata"
+
+# Setup mono
+monodir="$HOME/mono"
+monofile="$controlfolder/libs/mono-6.12.0.122-aarch64.squashfs"
+$ESUDO mkdir -p "$monodir"
+$ESUDO umount "$monofile" || true
+$ESUDO mount "$monofile" "$monodir"
+
+# Setup savedir linux
+$ESUDO rm -rf ~/.local/share/tiny_slash
+mkdir -p ~/.local/share
+mkdir -p "$GAMEDIR/savedata"
+ln -sfv "$GAMEDIR/savedata" ~/.local/share/tiny_slash
```

Comment:
Linking is done with `bind_directories` in the next line, just keep this line
```suggestion
mkdir -p "$GAMEDIR/savedata"
```